lib/environment.ts

```diff
@@ -1,15 +1,24 @@
 import fs from "fs";
 import fetch from "node-fetch";
 
+const maskToken = (token: string) =>
+  "*".repeat(Math.max(token.length-4, 0)) + token.slice(-4);
+
 export const getAuthToken = () => {
+  let token;
   if (process.env.VERCEL === "1") {
-    return process.env.AUTH_TOKEN || "";
+    token = process.env.SALEOR_AUTH_TOKEN || "";
   } else {
-    return fs.readFileSync(".auth_token", "utf8");
+    token = fs.readFileSync(".auth_token", "utf8");
```

Comment:
@magul `async` fs access would be slightly better here (let's do it after June 6)